### PR TITLE
clrcore: New methods for Ped and Vehicle class

### PR DESCRIPTION
**Pull request by pongo1231**

### CODE REVIEW

File: code/client/clrcore/External/Screen.cs [hunk 1]
```diff
@@ -327,7 +327,28 @@ private static PointF WorldToScreen(Vector3 position, float screenWidth, float s
...
+		/// Draws a text on the screen
+		/// </summary>
+		/// <param name="text">The text do draw</param>
+		/// <param name="x">The x offset between 0.0 and 1.0</param>
+		/// <param name="y">The y offset between 0.0 and 1.0</param>
+		/// <param name="scaleX">The x scale of the text</param>
+		/// <param name="scaleY">The y scale of the text</param>
+		/// <param name="color">The text color</param>
+		/// <param name="font">The font to use for the text</param>
+		public static void DrawText(string text, float x, float y, float scaleX, float scaleY, Color color, int font)
```
Comment (thers) [inline code/client/clrcore/External/Screen.cs]: Why? There's a `Text` class exactly for this purpose. https://github.com/citizenfx/fivem/blob/master/code/client/clrcore/External/Text.cs

Comment (pongo1231) [reply code/client/clrcore/External/Screen.cs]: Oops, I didn't see that one.

**Comment by vecchiotom** [comment]
+1 ~~would smash~~

File: code/client/clrcore/External/Screen.cs [hunk 2]
```diff
@@ -327,7 +327,8 @@ private static PointF WorldToScreen(Vector3 position, float screenWidth, float s
 			}
 
 			return new PointF(pointX * screenWidth, pointY * screenHeight);
-		}
+		}
```
Comment (thers) [inline code/client/clrcore/External/Screen.cs]: Hmmm

Comment (vecchiotom) [reply code/client/clrcore/External/Screen.cs]: noice.

Comment (thers) [reply code/client/clrcore/External/Screen.cs]: No.

**Comment by blattersturm** [comment]
:( please reopen once improved/fixed

**Pull request closed without merging**